independent-projects/resteasy-reactive/server/processor/src/main/java/org/jboss/resteasy/reactive/server/processor/ResteasyReactiveDeploymentManager.java

```diff
@@ -419,6 +419,7 @@ public Application get() {
             info.setApplicationPath(path);
             List<Closeable> closeTasks = new ArrayList<>();
             RuntimeDeploymentManager runtimeDeploymentManager = new RuntimeDeploymentManager(info, () -> executor,
+                    () -> executor,
```

Comment:
No lambdas in runtime code please 😉

Reply:
hey georgios, a lambda was already used (the first () -> executor), should I remove too and instantiate a supplier in the lines before ?
this piece of code is only used in ResteasyReactiveUnitTest it seems

Reply:
Yeah, please remove the existing one as well